app/views/settings/preferences/show.html.haml

```diff
@@ -7,6 +7,12 @@
   .fields-group
     = f.input :locale, collection: I18n.available_locales, wrapper: :with_label, include_blank: false, label_method: lambda { |locale| human_locale(locale) }
 
+    = f.input :setting_site_theme,
+      collection: ['', 'kkt'],
```

Comment:
I too was wondering where to put the definition of the themes.
I put it in app/helpers/settings_helper.rb, but anyway hardcoded...

It'll be great if themes list is created from the themes assets automatically in the future.